Client/src/Validation/validation.js

```diff
@@ -18,10 +18,17 @@ const registerValidation = joi.object({
       "string.empty": "Email is required",
     }),
 
-  password: joi.string().min(8).required().messages({
-    "string.min": "Password must be at least 8 characters",
-    "string.empty": "Password is required",
-  }),
+  password: joi
+    .string()
+    .min(8)
+    .pattern(/^(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/)
```

Comment:
As I mentioned in your other PR we can drop the 8 char requirement from the regex since it is already checked for by `.min(8)`.  No need for the redundant check